kustomize/v010-adapter/daemonset.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nri-plugin-v010-adapter
+spec:
+  template:
+    spec:
+      containers:
+        - name: plugin
+          image: "ghcr.io/containerd/nri/plugins/v010-adapter:unstable"
+          imagePullPolicy: Always
+          args:
+            - "-idx"
+            - "99"
+            - "-name"
+            - "v010-adapter"
+          volumeMounts:
+            - name: nri-socket
+              mountPath: /var/run/nri/nri.sock
+      volumes:
+      - name: nri-socket
```

Comment:
This would also require additional bits from the host for the original hookesque v0.1.0 NRI interface to work. Those would be by default the config file `/etc/nri/conf.json` and the old plugin binary directory `/opt/nri/bin`.